packages/solana-intents/src/key_value.rs

```diff
@@ -55,14 +55,20 @@ where
             alt((
                 delimited(
                     tag(" "),
-                    take_till1(|c: <I as Input>::Item| c.is_newline()),
+                    take_till1(|c: <I as Input>::Item| c.is_newline() || c.as_char() == '\r'),
```

Comment:
this was inconsistent with line_ending, with checks both` /r/n` and `/n`

Reply:
I think we can use `not_line_ending` here instead of this line, e.g.

```rs
delimited(tag(" "), not_line_ending, alt((line_ending, eof)))
```

Reply:
https://github.com/fogo-foundation/fogo-sessions/pull/204